src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/UpsertSearchParamsWithOptimisticConcurrency.sql

```diff
@@ -0,0 +1,88 @@
+﻿/*************************************************************
+    Stored procedures - UpsertSearchParams_2
```

Comment:
Please remove all the text before CREATE PROCEDURE. This meaningless text is prepended in all system tables, so it is hard to debug.